addons/base_vat/models/res_partner.py

```diff
@@ -67,42 +17,51 @@
 _eu_country_vat_inverse = {v: k for k, v in _eu_country_vat.items()}
 
 _ref_vat = {
+    'al': 'ALJ91402501L',
+    'ar': 'AR200-5536168-2 or 20055361682',
     'at': 'ATU12345675',
     'be': 'BE0477472701',
     'bg': 'BG1234567892',
     'ch': 'CHE-123.456.788 TVA or CH TVA 123456',  # Swiss by Yannick Vaucher @ Camptocamp
     'cl': 'CL76086428-5',
     'co': 'CO213123432-1 or CO213.123.432-1',
-    'cy': 'CY12345678F',
+    'cy': 'CY10259033P',
     'cz': 'CZ12345679',
     'de': 'DE123456788',
     'dk': 'DK12345674',
+    'do': 'DO1-01-85004-3 or 101850043',
+    'ec': 'EC1792060346-001',
     'ee': 'EE123456780',
     'el': 'EL12345670',
     'es': 'ESA12345674',
     'fi': 'FI12345671',
-    'fr': 'FR32123456789',
+    'fr': 'FR23334175221',
     'gb': 'GB123456782',
     'gr': 'GR12345670',
     'hu': 'HU12345676',
     'hr': 'HR01234567896',  # Croatia, contributed by Milan Tribuson
     'ie': 'IE1234567FA',
+    'is': 'IS062199',
     'it': 'IT12345670017',
     'lt': 'LT123456715',
     'lu': 'LU12345613',
     'lv': 'LV41234567891',
+    'mc': 'FR53000004605',
     'mt': 'MT12345634',
-    'mx': 'ABC123456T1B',
+    'mx': 'MXGODE561231GR8 or GODE561231GR8',
     'nl': 'NL123456782B90',
     'no': 'NO123456785',
     'pe': 'PER10254824220 or PED10254824220',
     'pl': 'PL1234567883',
     'pt': 'PT123456789',
     'ro': 'RO1234567897',
+    'rs': 'RS101134702',
+    'ru': 'RU123456789047',
```

Comment:
@hpr-odoo From where did you get these changes?

Reply:
@jco-odoo from stdnum library